MapboxNavigation/CarPlayMapViewController.swift

```diff
@@ -93,6 +88,34 @@ class CarPlayMapViewController: UIViewController, MGLMapViewDelegate {
             mapView.localizeLabels()
         }
     }
+    
+    func resetCamera(animated: Bool = false, defaultAltitude: Bool = false) {
+        let camera = mapView.camera
+        if defaultAltitude {
+            camera.altitude = 16000
```

Comment:
Replace `defaultAltitude` with an `altitude` argument that defaults to 16,000 instead of false.